app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = Settings.reg.email
+  before_save { self.email = email.downcase }
```

Comment:
```suggestion
  before_save { email.downcase! }
```
phân biệt giúp anh, downcase! khác gì downcase